packages/nodejs/src/client.ts

```diff
@@ -55,7 +55,12 @@ export class BaseClient implements Client {
    * Creates a new instance of the `Appsignal` object
    */
   constructor(options: Partial<AppsignalOptions> = {}) {
-    const { ignoreInstrumentation } = options
+    const {
+      instrumentRedis,
+      instrumentHttp,
+      instrumentHttps,
+      instrumentPg
+    } = options
 
     this.config = new Configuration(options)
```

Comment:
If these configuration options are read directly from the options passed to the constructor, changing their value using environment variables will have no effect. (This bug was already present on the existing `ignoreInstrumentation` implementation)

That is, as it's currently implemented, setting any of the `APPSIGNAL_INSTRUMENT_*` environment variables to `false` won't actually disable that instrumentation.

For the environment variable values to be taken into account as well, these options should be read from the Configuration object, after it is initialised.

```suggestion
    this.config = new Configuration(options)
    
    const {
      instrumentRedis,
      instrumentHttp,
      instrumentHttps,
      instrumentPg
    } = this.config.data
```